src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -555,6 +565,7 @@ bool AppHost::HasWindow()
 void AppHost::_DispatchCommandline(winrt::Windows::Foundation::IInspectable /*sender*/,
                                    Remoting::CommandlineArgs args)
 {
+    // TODO! Should we summon the window whenever we dispatch a commandline to it? methinks yes, but not in this PR.

```

Comment:
bare todo

Reply:
I vote yes

Reply:
@DHowett for the sanity check too, because I also vote yes

Reply:
I vote yes.